<launch>
    <arg name="gh_ip" default="192.168.0.100" />
    <arg name="ur_ip" default="192.168.0.10" />
    <arg name="ros_rate" default="10" />
    <arg name="gh_tick_interval" default="333" />
    <node name="rosbridge_server" pkg="rosbridge_server" type="rosbridge_websocket" />
    <node name="db_runtime" pkg="deepbuilder" type="runtime" output="screen" args="$(arg gh_ip) $(arg ros_rate)"/>
    <node name="db_learner" pkg="deepbuilder" type="learner.py" output="screen"/>
    <!--node name="db_robot" pkg="deepbuilder" type="robot_control.py" output="screen" args="$(arg ur_ip)"/-->
    <node name="db_gripper" pkg="deepbuilder" type="gripper_control" output="screen" />
    <node name="rviz" pkg="rviz" type="rviz" />
</launch>
